x/paramfilter/types.go

```diff
@@ -0,0 +1,15 @@
+package paramfilter
+
+import (
+	sdkerrors "cosmossdk.io/errors"
+)
+
+const (
+	// ModuleName is the name of the module
+	ModuleName    = "paramfilter"
+	baseErrorCode = 91710
+)
+
+// ErrBlockedParameter is the error wrapped when a proposal to change a
+// blocked parameter is submitted.
+var ErrBlockedParameter = sdkerrors.Register(ModuleName, baseErrorCode, "blocked parameter change, hard fork required")
```

Comment:
```suggestion
var ErrBlockedParameter = sdkerrors.Register(ModuleName, baseErrorCode, "parameter can not be modified")
```
nit: I think the words `hard fork required` could spook a few users